spec/validators/claim/advocate_hardship_claim_validator_spec.rb

```diff
@@ -7,7 +7,14 @@
 
   let(:claim) { create(:advocate_hardship_claim) }
 
-  before { seed_fee_schemes }
+  before(:all) do
+    seed_fee_schemes
+    seed_fee_types
+  end
+
+  after(:all) do
+    clean_database
+  end
```

Comment:
This is used in a few places so implies we can either use a [shared context](https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-context)or make part of the example. This would also mean we could change this mechanism in one place if we wanted to change it for performance reasons

Reply:
👍 have amended to not seed fee_types or use the before/after(:all) blocks